content/blog/overview-of-the-platform-level-data-model-for-redfish®-device-enablement-standard.md

```diff
@@ -0,0 +1,158 @@
+* Reboot required to load new device configuration settings
+
+## Operational model with PLDM for RDE
+
+The implementation of PLDM for RDE standard, in both iLO and devices, changes the iLO role from "active" to "pass-through". In this configuration, the iLO receives HTTP requests from Redfish clients and translates then into a set of RDE requests to the device. When the PLDM dialog between the iLO and the device is over, the iLO sends back a final HTTP response to the Redfish client (See picture below)
+
+![Operational model with PLDM for RDE](/img/withpldmrde.png "Operational model with PLDM for RDE")
+
+In this model, the Management Controller and device firmware don't need to be aligned because the MC does not contain any device properties. It contains only the necessary information to translate the Redfish client requests into PLDM for RDE messages.
+
+As a result, if all the computer makers implement the PLDM for RDE protocol suite in their Redfish service, the number of communication protocols to be implemented and maintained by the device providers is reduced to one.
+
+Lastly, PLDM for RDE device modifications do not require a reboot of the server to become effective in the device. However, a reboot may be required by the operating system or by applications.
+
+## HPE implementation of PLDM for RDE
+
+On the server side, the implementation of PLDM for RDE started in iLO 5 [firmware version 2.33](https://hewlettpackard.github.io/ilo-rest-api-docs/ilo5/#changelog) and newer firmware releases implement more and more PLDM related features that can be leveraged by devices providing their support of this standard. As an example, Redfish clients connected to an iLO 5 version 2.65 can perform read and write operations on Logical Drives hosted by  [HPE SR Smart Arrays](https://www.hpe.com/psnow/doc/a00047736enw.html?jumpid=in_pdp-psnow-qs) running firmware version 5.00.
+
+This statement is clearly mentioned in the `readme.txt` file embedded in the `ilo5_265.fwpkg` firmware package. You can use the `unzip` utility to extract this file from the package as shown in the next picture.
+
+![RDE support in iLO 5 firmware version 2.65](/img/rdesupportinfw265.png "RDE support in iLO 5 firmware version 2.65")
+
+
+
+### Storage controllers
+
+The [Enhancements paragraph](https://support.hpe.com/connect/s/softwaredetails?language=en_US&softwareId=MTX_a06269a339da44ada286007600&tab=Enhancements) of HPE Smart Arrays version 5.00 mentions several new PLDM and DMTF features for this version (see picture below).
+
+![PLDM Enhancements](/img/firmwareenhancements.png "PLDM Enhancements")
+
+
+
+If you power on a server containing this type of storage controller and the correct firmware versions, you will notice a device with string `DE` in its URI similar to : `/redfish/v1/systems/1/DExxxxxx`. This `DE` string is synonym of a device supporting the PLDM for RDE standard and discovered as such by the Redfish service.
```

Comment:
Change "This 'DE' string is synonym of a device..." to "This 'DE' string is synonymous with a device"